docs/1.0. Certificate Provisioning.md

```diff
@@ -221,126 +225,128 @@ If an EST Client implements the AMWA IS-04 Node API, then the EST Client SHOULD
+Renewal of the Root CA SHOULD be attempted no sooner than 50% of the certificate's expiry time. It is RECOMMENDED that certificate renewal is performed after 80% of the expiry time. To renew the Root CA and the EST Client's TLS Certificate, follow the [Initial Certificate Provisioning](#initial-certificate-provisioning) workflow, renewing both the Root CA and server certificates in the process.
 
-If the returned Root Certificate Authority by the EST Server is the same as the existing Root Certificate Authority, the EST Client SHOULD re-attempt renewal of the Root Certificate Authority after half of the remaining period of validity has elapsed.
+If the returned Root Certificate Authority by the EST Server is the same as the existing Root Certificate Authority, the EST Client SHOULD reattempt renewal of the Root Certificate Authority after half of the remaining period of validity has elapsed.
 
-### EST Server Side Key Generation
+### EST Server-Side Key Generation
 
-An EST Server MAY support server side key generation, allowing devices to request a TLS Certificate without having to perform the computationally expensive process of generating a private key.
+An EST Server MAY support server-side key generation, allowing devices to request a TLS Certificate without having to perform the computationally expensive process of generating a private key.
 
-If the EST Server supports server side key generation, the EST Client MAY make a request to the EST Server endpoint `/serverkeygen` in accordance with [RFC 7030 - Section 4.4](https://tools.ietf.org/html/rfc7030#section-4.4).
+If the EST Server supports server-side key generation, the EST Client MAY make a request to the EST Server endpoint `/serverkeygen` in accordance with [RFC 7030 - Section 4.4](https://tools.ietf.org/html/rfc7030#section-4.4).
 
 The EST Client SHOULD request that the returned private key is encrypted using either symmetric or asymmetric encryption, by including the appropriate information in the CSR.
 
-The EST Client SHOULD include the manufacturer installed TLS Client Certificate if present and valid during the TLS handshake with the EST Server. If the TLS certificate is no longer valid, the [Expired Manufacturer Issued TLS Client Certificate](#Expired-Manufacturer-Issued-TLS-Client-Certificate) workflow should be followed.
+The EST Client SHOULD include the manufacturer installed TLS Client Certificate if present and valid during the TLS handshake with the EST Server. If the TLS certificate is no longer valid, the [Expired Manufacturer-Issued TLS Client Certificate](#Expired-Manufacturer-Issued-TLS-Client-Certificate) workflow should be followed.
 
 EST Client SHOULD handle the EST Server's response as per [Certificate Request Response](#certificate-request-response).
```

Comment:
```suggestion
The EST Client SHOULD include the manufacturer installed TLS Client Certificate if present and valid during the TLS handshake with the EST Server. If the TLS certificate is no longer valid, the [Expired Manufacturer-Issued TLS Client Certificate](#expired-manufacturer-issued-tls-client-certificate) workflow should be followed.

EST Client SHOULD handle the EST Server's response as per [Certificate Request Response](#certificate-request-response).
```

Reply:
I think we already did that in c01a800804603b46331cacfe22dde6765b7cf112...